go/vt/vttablet/tabletserver/txthrottler/tx_throttler_test.go

```diff
@@ -128,9 +132,13 @@ func TestEnabledThrottler(t *testing.T) {
 	// This call should not be forwarded to the go/vt/throttler.Throttler object.
 	throttler.state.StatsUpdate(rdonlyTabletStats)
 	// The second throttle call should reject.
-	assert.True(t, throttler.Throttle())
+	assert.True(t, throttler.Throttle(0))
 	assert.Equal(t, int64(2), throttler.requestsTotal.Get())
 	assert.Equal(t, int64(1), throttler.requestsThrottled.Get())
+	// This call should not reject due to criticality
+	if result := throttler.Throttle(100); result != false {
+		t.Errorf("want: false, got: %v", result)
+	}
```

Comment:
use assert then `t.errorf` 
also check that the counters are not changed.

Reply:
Fixed.